experimental/Schemes/IdealSheaves.jl

```diff
@@ -639,6 +632,19 @@ function minimal_associated_points(I::IdealSheaf)
   return associated_primes_result
 end
 
+@doc raw"""
+    associated_points(I::IdealSheaf)
+
+Returns a `Vector` of `IdealSheaf`s corresponding to the associated points of ``I`` on ``scheme(I)``.
+
+Note:
+For usability reasons an associated points are not encoded as a subscheme, but as the corresponding ideal sheaf defining the subscheme.
```

Comment:
```suggestion
For usability reasons associated an point is not encoded as a subscheme, but as the corresponding ideal sheaf defining the subscheme.
```

Reply:
"associated an point" still doesn't make sense to me. Perhaps you meant this?
```suggestion
For usability reasons an associated point is not encoded as subscheme, but as the corresponding ideal sheaf defining the subscheme.
```
or
```suggestion
For usability reasons associated points are not encoded as subschemes, but as the corresponding ideal sheaves defining the subschemes.
```